plutus-core/plutus-core/src/PlutusCore/Default/Universe.hs

```diff
@@ -121,14 +119,14 @@ instance GShow DefaultUni where gshowsPrec = showsPrec
 instance Show (DefaultUni a) where
     show DefaultUniInteger             = "integer"
     show DefaultUniByteString          = "bytestring"
-    show DefaultUniChar                = "char"
+    show DefaultUniText                = "text"
     show DefaultUniUnit                = "unit"
     show DefaultUniBool                = "bool"
     show DefaultUniProtoList           = "list"
     show DefaultUniProtoPair           = "pair"
     show (uniF `DefaultUniApply` uniB) = case uniF of
         DefaultUniProtoList -> case uniB of
-            DefaultUniChar -> "string"
+            DefaultUniText -> "string"
```

Comment:
remove, this was for handling lists of chars

Reply:
```suggestion

```